quactography/visu/gs_square_loss_for_p.py

```diff
@@ -24,42 +25,43 @@ def visualize_optimal_paths_edge_rep(
     Returns
     -------
     None """
-    reps = []
-    alphas = []
-    square_loss = []
-    param_count = 0
+    delta_dict = defaultdict(list)
     path = Path(in_folder)
 
     glob_path = path.glob('*.npz')
 
     for in_file_path in glob_path:
-        _, _, min_cost, h, _, rep, _ = load_optimization_results(in_file_path)
+        _, dist_prob, min_cost, h, _, rep, _ = load_optimization_results(in_file_path)

```

Comment:
Add comments, for outsiders it is very tough to understand what is going on in this function...